core/Dockerfile

```diff
@@ -13,17 +13,11 @@ WORKDIR /build/questdb
 
 RUN ../apache-maven-3.6.3/bin/mvn clean package -DskipTests -P build-web-console,build-binaries,use-built-in-nodejs
 
-WORKDIR /build/questdb/core/target
+FROM openjdk:16-alpine
```

Comment:
using generic image overinflates the image size:

![image](https://user-images.githubusercontent.com/7276403/111171183-95d3a500-859c-11eb-8625-aaef03c4a70f.png)